### ISSUE TITLE
On Microsoft's SAW device, Explorer eats the leading 'h' is when submitting SAS URI to connect to Storage accounts

### ISSUE THREAD

**Issue by arorasam** [microsoft/AzureStorageExplorer#3497]
Hello,

On my SAW device, whenever I submit a SAS token URI to connect to Storage, the initial 'h' of https:// gets swallowed, resulting in exception stack below. Same URI works fine on my non-SAW desktop,laptop.
Azure Storage Explorer is 1.12.0, AzCopy version is 10.3.3
What's also a problem is upgrading Azure Storage Explorer on SAW. Whenever I attempt to upgrade, I don't see any activity or progress - whether I update right away, on close or from the Software Center. I do see about 6 instances of Storage Explorer running in task manager. Only on killing them can I get a working instance again. 
Due to this, am unable to upgrade either - and this makes live site very difficult!

Here's a redacted example URI that doesn't work on SAW-Storage Explorer but worked fine on desktop: https://REDACTED.file.core.windows.net/d6c5a14a89a94ed3bc854674f43d2465?sv-REDACTED-sp=rl

If a failing real URL will help debugging, please IM me over Teams from within corp network. 

**Exception Stack**
```
{
  "name": "URIError",
  "message": "The provided URI \"ttps:wasd2prodauea1abake50.file.core.windows.net\" is invalid.",
  "__zone_symbol__error": {
    "originalStack": "Error: The provided URI \"ttps:wasd2prodauea1abake50.file.core.windows.net\" is invalid.\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)",
    "zoneAwareStack": "Error: The provided URI \"ttps:wasd2prodauea1abake50.file.core.windows.net\" is invalid.\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)"
  },
  "stack": "URIError: The provided URI \"ttps:wasd2prodauea1abake50.file.core.windows.net\" is invalid.\n    at exports.isValidHost (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\util\\validate.js:101:13)\n    at Array.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\servicesettings.js:138:25)\n    at Object.exports.matchedSpecification (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\servicesettings.js:251:54)\n    at Function.StorageServiceSettings.createFromSettings (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\storageservicesettings.js:250:34)\n    at Function.StorageServiceSettings.createFromConnectionString (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\storageservicesettings.js:125:35)\n    at Function.StorageServiceClient.getStorageSettings (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\storageserviceclient.js:919:53)\n    at new FileService (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\services\\file\\fileservice.core.js:93:53)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\Azure\\AzureStorage\\Files\\FileUtilities.js:45:43\n    at Function.promise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\q\\q.js:702:9)\n    at Object.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\Azure\\AzureStorage\\Files\\FileUtilities.js:31:49)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at fulfilled (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:107:62)\n    at ZoneDelegate.invoke (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:242:26)\n    at Zone.run (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:113:43)",
  "originalStack": "Error: The provided URI \"ttps:wasd2prodauea1abake50.file.core.windows.net\" is invalid.\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)",
  "zoneAwareStack": "Error: The provided URI \"ttps:wasd2prodauea1abake50.file.core.windows.net\" is invalid.\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)",
  "__zone_symbol__stack": "URIError: The provided URI \"ttps:wasd2prodauea1abake50.file.core.windows.net\" is invalid.\n    at exports.isValidHost (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\util\\validate.js:101:13)\n    at Array.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\servicesettings.js:138:25)\n    at Object.exports.matchedSpecification (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\servicesettings.js:251:54)\n    at Function.StorageServiceSettings.createFromSettings (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\storageservicesettings.js:250:34)\n    at Function.StorageServiceSettings.createFromConnectionString (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\storageservicesettings.js:125:35)\n    at Function.StorageServiceClient.getStorageSettings (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\common\\services\\storageserviceclient.js:919:53)\n    at new FileService (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\azure-storage\\lib\\services\\file\\fileservice.core.js:93:53)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\Azure\\AzureStorage\\Files\\FileUtilities.js:45:43\n    at Function.promise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\q\\q.js:702:9)\n    at Object.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\Azure\\AzureStorage\\Files\\FileUtilities.js:31:49)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at fulfilled (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:107:62)\n    at ZoneDelegate.invoke (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:242:26)\n    at Zone.run (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:113:43)",
  "__zone_symbol__name": "URIError",
  "__zone_symbol__message": "The provided URI \"ttps:wasd2prodauea1abake50.file.core.windows.net\" is invalid.",
  "__zone_symbol__originalStack": "Error: The provided URI \"ttps:wasd2prodauea1abake50.file.core.windows.net\" is invalid.\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)",
  "__zone_symbol__zoneAwareStack": "Error: The provided URI \"ttps:wasd2prodauea1abake50.file.core.windows.net\" is invalid.\n    at AppInsightsAsyncCorrelatedErrorWrapper.ZoneAwareError (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:811:33)\n    at new AppInsightsAsyncCorrelatedErrorWrapper (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\applicationinsights\\out\\AutoCollection\\CorrelationContextManager.js:193:18)\n    at Function.ExceptionSerializer.deserialize (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\se-exception-serializer\\dist\\ExceptionSerializer.js:49:28)\n    at CloudExplorerProxyMarshaler._finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:83:91)\n    at Object.finalizeOperation (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\CloudExplorerProxyMarshaler.js:107:80)\n    at Function.<anonymous> (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:28:63)\n    at step (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:136:27)\n    at Object.next (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:117:57)\n    at C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:110:75\n    at new ZoneAwarePromise (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\zone.js\\dist\\zone-node.js:551:29)\n    at Object.__awaiter (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\tslib\\tslib.js:106:16)\n    at JsonPortMarshaler.standardMarshaler.(anonymous function) [as finalizeOperation] (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\out\\app\\renderer\\marshalers\\StandardMarshaler.js:22:32)\n    at JsonPortMarshaler.onMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\marshaler.js:101:41)\n    at Port.EventTarget.dispatchEvent (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\common\\event.js:69:30)\n    at Port.processMessage (C:\\Program Files (x86)\\Microsoft Azure Storage Explorer\\resources\\app\\node_modules\\daytona\\pluginHost\\messaging\\communication.js:163:18)"
}
```

**Comment by MRayermannMSFT** [comment]
@arorasam I just tried reproducing this while on a SAW and I can't get the behavior you described. I tried:
1. Using your example SAS URI
2. Using a freshly generated blob SAS URI
3. Using a freshly generated file SAS URI

- Where are you generating the SAS URI from?
- If you past the SAS URI into something like notepad is it correct?
- If you copy and paste a correct SAS URI from notepad to Storage Explorer does this repro?
- Does this happen consistently?
- Does it only happen with a certain account or container?
